receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -269,15 +268,23 @@ func addHistogramDatapoints(_ pmetric.ResourceMetrics, _ labels.Labels, _ writev
 // addDatapoints adds the labels to the datapoints attributes.
 // TODO: We're still not handling several fields that make a datapoint complete, e.g. StartTimestamp,
 // Timestamp, Value, etc.
```

Comment:
```suggestion
// TODO: We're still not the StartTimestamp.
```

Let's also clean up the TODO comment :)